requirements-dev.txt

```diff
@@ -5,3 +5,7 @@ pep8
 nosexcover
 nose
 selenium
+bs4
```

Comment:
It should not needed to repeat these here  - the first line of this file already imports everything from requirements.txt

Reply:
OK but it doesnt. The first line is commented so it doesn't install what's in requirements.txt.

Am i right?

Reply:
So I uncommented the first line in requirements-dev.txt.
Then I built an image from Dockerfile.dev with the command:
`docker build -f Dockerfile.dev -t "pat-osm-reporter" .`

And it crashed as expected: maximum recursion depth exceeded.
Because we ADD HOST/requirements-dev.txt as CONTAINER/requirements.txt
and RUN the pip install command which will install from requirements.txt => infinite loop, maximum recursion depth! KABOOM.

Does it make sense?